learn/core_concepts/relevancy.md

```diff
@@ -106,15 +113,15 @@ The `attribute` rule sorts the results by [attribute importance](/learn/core_con
 
 ### Exactness
 
-`Knight Moves` is displayed before `Knights of Badassdom`. `Knight` is exactly the same as the search query `Knight` whereas there is a letter of difference between `Knights` and the search query `Knight`.
+`Knight Moves` is displayed before `Knights of Badassdom`. `Knight` is exactly the same as the search query `Knight` whereas, there is a letter of difference between `Knights` and the search query `Knight`.
```

Comment:
That new comma sounds really weird. It separates the conjunction from the thing it is supposed to connect to the previous clause. If you really want to, you could add a comma before "whereas" (e.g. `search query 'Knight', whereas there is`).

```suggestion
`Knight Moves` is displayed before `Knights of Badassdom`. `Knight` is exactly the same as the search query `Knight` whereas there is a letter of difference between `Knights` and the search query `Knight`.
```